src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -193,7 +196,7 @@ internal static class DefinitionItemExtensions
             var documentSpan = await ClassifiedSpansAndHighlightSpanFactory.GetClassifiedDocumentSpanAsync(
                 document, sourceSpan, cancellationToken).ConfigureAwait(false);
 
-            return new SourceReferenceItem(definitionItem, documentSpan, referenceLocation.SymbolUsageInfo);
+            return new SourceReferenceItem(definitionItem, documentSpan, referenceLocation.SymbolUsageInfo, referenceLocation.AdditionalProperties);
```

Comment:
The property name here should be renamed to `FindUsagesProperties` or `AdditionalFindUsagesProperties`